.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Create Release
```

Comment:
What's the difference between release and publish? don't they go hand and hand? < thinkingfaceemoji />

Reply:
Ha, I guess here "release" is a "thing",  <https://github.com/NASA-Openscapes/edlfs/releases>, which you can do stuff with like create/publish/etc.